src/test/java/teammates/test/cases/ui/StudentProfileEditSaveActionTest.java

```diff
@@ -8,12 +8,17 @@
 
 import teammates.common.datatransfer.AccountAttributes;
 import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.StudentAttributes;
 import teammates.common.datatransfer.StudentProfileAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
 import teammates.common.util.Const;
 import teammates.common.util.FieldValidator;
 import teammates.common.util.StringHelper;
 import teammates.test.driver.AssertHelper;
 import teammates.ui.controller.RedirectResult;
+import teammates.ui.controller.ShowPageResult;
+import teammates.ui.controller.StudentCourseDetailsPageAction;

```

Comment:
@chowyb's suggestion is using UI tests, not action tests. It is not right to have `StudentCourseDetailsPageAction` appearing in an action test not related to it. If you still want to test via action test, go to `StudentCourseDetailsPageActionTest`, upload a profile picture before the test is executed (e.g at `@BeforeClass`), and test accordingly.